demo/main.js

```diff
@@ -1227,9 +1227,8 @@ shakaDemo.Main = class {
       // Remove all not-player-applied configurations, by resetting the
       // configuration then re-applying the desired configuration.
       this.player_.resetConfiguration();
-      this.player_.configure(this.desiredConfig_);
+      this.readHash_();
```

Comment:
So I did some practical testing, and couldn't find any evidence that this PR leads to the hash or desiredConfig_ values changing.
It's possible that comment below was true at some earlier point in the demo app's development...
Anyway, this is probably safe.